ext/rlibmemcached/rlibmemcached.i

```diff
@@ -175,7 +175,7 @@ VALUE memcached_fetch_rvalue(memcached_st *ptr, const char **key, size_t *key_le
   *error = MEMCACHED_TIMEOUT; // timeouts leave error uninitialized
   memcached_result_st *result = memcached_fetch_result(ptr, &ptr->result, error);
   VALUE str = Qnil;
-  if (result == NULL || *error != MEMCACHED_SUCCESS) {
+  if (result == NULL) {
```

Comment:
This change was intended to test out whether we can get additional info out of the fetch calls. I haven't yet seen any indication that this is helpful or harmful, the test suite passes, but this shouldn't be material to the change that allows us to consistently raise errors. And dropping this change would clear a _lot_ of the diff because it require re-SWIG-ing.